src/base/PolymeshTransactionBase.ts

```diff
@@ -712,7 +712,7 @@ export abstract class PolymeshTransactionBase<
    *
    * @note this depends on the type of transaction itself (e.g. `staking.bond` can't be subsidized, but `asset.createAsset` can)
    */
-  public abstract supportsSubsidy(): boolean;
+  public abstract assertTransactionSupportsSubsidy(): void;
```

Comment:
since i renamed this, do you think @polymath-eric  this should be marked as breaking change? I'm not sure end users have been using this method for explicitly checking if the transactions supports subsidy or not?

Reply:
I think `supportsSubsidy` should be left as it is.

Internally assert methods are convenient, but for a user they aren't so nice to use for checking things like this.

If you want an `assertTransactionSupportsSubsidy` make it `protected` and have it call `supportsSubsidy`

Reply:
Regardless of if people are using it or not, if a method is in a public API gets changed, then its a breaking change. Better to avoid it now imo.